generator/AWSPSGeneratorLib/HelpMaterials/Examples/LM/New-LMAlias.xml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$routingConfigValues = @{}
+$routingConfigValues.Add("1",0.6)
+New-LMAlias -FunctionName "MylambdaFunction123" -RoutingConfig_AdditionalVersionWeight $routingConfigValues -Description "Alias for version 4" -FunctionVersion 4 -Name "PowershellAlias"</code>
```

Comment:
Since it is simple hash-table, we could add inline
`New-LMAlias -FunctionName "MylambdaFunction123" -RoutingConfig_AdditionalVersionWeight @{Name="1";Value="0.6}`

Reply:
Made the changes.